File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SeekingGlass.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import androidx.work.WorkRequest;
import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class SeekingGlass extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_seeking_glass_name;
        this.idDescription = R.string.accessory_seeking_glass_description;
        this.idEffect = R.string.accessory_seeking_glass_effect;
        this.idImage = R.drawable.seeking_glass;
        this.source.add(Integer.valueOf(R.string.raid_name_the_slime_pond));
        this.price = WorkRequest.MIN_BACKOFF_MILLIS;
        this.alwaysHits = true;
        this.intelligence = 25;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/SeekingGlass.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class SeekingGlass : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_seeking_glass_name
        idDescription = R.string.accessory_seeking_glass_description
        idEffect = R.string.accessory_seeking_glass_effect
        idImage = R.drawable.seeking_glass
        source.add(R.string.raid_name_the_slime_pond)
        price = 10000L
        alwaysHits = true
        intelligence = 25
    }
}
